src/tests/routes/me/tokens/snapshots/all__routes__me__tokens__create__create_token_success.snap

```diff
@@ -1,13 +1,14 @@
 ---
 source: src/tests/routes/me/tokens/create.rs
-expression: response.into_json()
+expression: response.json()
```

Comment:
any idea where these changes are coming from? I don't see these expressions change in the diff here 🤔

Reply:
This change was generated by the cargo-insta.

Reply:
My insta version:
```console
❯ cargo insta --version
cargo-insta 1.37.0
```

Reply:
All reverted.